packages/util/src/url.ts

```diff
@@ -0,0 +1,3 @@
+export function isCloudWorkstation(url: string) {
+  return url.endsWith('cloudworkstations.dev');
```

Comment:
let's add the explicit \. before the domain to make sure other domains like amazoncloudworkstations.dev don't trigger this